alpha/external-web/test/wcag/test-wcag-aa.js

```diff
@@ -0,0 +1,28 @@
+var fs = require('fs')
+var pa11y = require('pa11y')
```

Comment:
The `pa11y` library has a dependency on `PhantomJS` this will cause an `npm start` command to fail unless the user happens to have `PhantomJS` installed locally. 

Better to separate this out into a module in a similar manner to the `Phantomas` (perf-budget module).